packages/conventional-changelog/index.js

```diff
@@ -6,7 +6,16 @@ function conventionalChangelog(options, context, gitRawCommitsOpts, parserOpts,
 
   if (options.preset) {
     try {
-      options.config = require('conventional-changelog-' + options.preset.toLowerCase());
+      var scope = '';
+      var name = options.preset.toLowerCase();
```

Comment:
There could be upper-case preset packages (At one point `npm` allowed upper case package names, so `npm info JSON` is not the same as `npm info json`).

I would just respect the casing provided by the user.

Reply:
The `toLowerCase` is existing code so that seems a bit out of scope for this PR

Reply:
> The toLowerCase is existing code so that seems a bit out of scope for this PR

That's a good point. It would be out of scope.